extensions/2.0/Vendor/NV_materials_mdl/README.md

```diff
@@ -0,0 +1,511 @@
+
+Since the MDL material replaces the standard glTF material, it is highly recommended that authoring applications also provide a closely matching standard glTF PBR material next to an MDL material such that implementations that do not understand this extension can nonetheless show a representative rendering of this glTF file. Defining a fallback material is, however, optional. The [best practices](#best-practices) section provides more details on this matter. 
+
+MDL materials are defined through functions in the language that return the built-in type `material`. This extension operates on function calls of functions defined in referenced MDL modules. These function calls are defined in the `functionCalls` array property which is defined in the top-level property `extensions.NV_materials_mdl`. Modules which are referenced by the function calls and types are defined in the `modules` array. Lastly, BSDF measurement resources are defined in the `bsdfMeasurements` array.
+
+```json
+"extensions": {
+    "NV_materials_mdl": {
+        "modules": [
+            ...
+        ],
+        "bsdfMeasurements": [
+            ...
+        ],
+        "functionCalls": [
+            ...
+        ]
+    }
+}
+```
+
+## Adding a MDL Material to an Existing Material
+MDL materials must be referenced in the material by specifying an index into the `functionCalls` array using the `funcionCall` property within a material's `extensions.NV_materials_mdl` property. This function call index represents the root node of a graph of function calls and must have the built-in return type `material`.
+
+```json
+"materials": [
+    {
+        "extensions": {
+            "NV_materials_mdl": {
+                "functionCall": 1
+            }
+        }
+    }
+]
+```
+
+## Defining MDL Modules
+All modules that are referenced by the function calls in the `functionCalls` array must be listed in the `modules` array. A `module` has the following properties:
+
+| Property | Type | Description | Required |
+|:---------|:-----|:------------|:---------|
+| `uri` | `string` | The URI (or IRI) of the MDL module. | No |
+| `bufferView` | `integer` | The index of the bufferView that contains the MDL module. | No |
```

Comment:
> The index of the bufferView

Please use plain text when not using formatting, i.e.:
>The index of the buffer view

Also in the text below.